testing/tests/DevExpress.core/utils.position.tests.js

```diff
@@ -0,0 +1,31 @@
+import config from 'core/config';
```

Comment:
Let's merge this file with this one:
https://github.com/DevExpress/DevExtreme/blob/fefd39b7ef1acf829100378d8aec47ea384f1bce/testing/tests/DevExpress.animation/position.tests.js

Reply:
I don't think we should do that. The positioning utilities located in `animation/utils` are used to support the animation module.
This one is private utility, used to get start direction or [base alignment](https://github.com/DevExpress/DevExtreme/blob/20_1/js/ui/data_grid/ui.data_grid.export.js#L113) based on passed argument or global `rtlEnabled` config.

Reply:
Thanks! My mistake...